tests/Integration/PrestaShopBundle/Controller/Sell/Catalog/ProductControllerTest.php

```diff
@@ -66,15 +60,13 @@ public function setUp(): void
             );
             $this->changedProductFeatureFlag = true;
         }
-
-        parent::setUp();
     }
 
     public function tearDown(): void
     {
         parent::tearDown();
         if ($this->changedProductFeatureFlag) {
-            $featureFlagModifier = $this->client->getContainer()->get('prestashop.core.feature_flags.modifier');
+            $featureFlagModifier = self::$kernel->getContainer()->get('prestashop.core.feature_flags.modifier');
```

Comment:
@jolelievre why? 🤔

Reply:
Because I keep getting an exception when the tests are run, `$this->client->getContainer()` returns null I don't know why I don't even know why tearDown is run before even the slightest test has started..

Maybe an issue related to processIsolation again... But anyway this doesn't work so I'll have to find another solution but I can't reproduce the bug locally 😅